### ISSUE TITLE
Dynmap Weblogin doesnt work in builds higher then build 62 (1.20.1)

### ISSUE THREAD

**Issue by CloudeLecaw** [PaperSpigot/Paper#9457]
### Expected behavior

Dynmap weblogin should work... and not fails to login

### Observed/Actual behavior

On your dynmap webpage your normaly can login with the door button if your registered
![Snag_5e42eb](https://github.com/webbukkit/dynmap/assets/12080017/afdc97e5-6cea-44c3-99b3-3fabb7c58d30)

![Snag_59fdf6](https://github.com/webbukkit/dynmap/assets/12080017/888730c8-21fd-4f40-ade5-e8e4a61a3c60)

We noticed that trying to login there fails. Your also get a message that your login failed.
We took a while befor noticed its because of a newer build of paper 1.20.1

* **Dynmap Version:** *Dynmap_3.6-beta2*
* **Server Version:** *paper 1.20.1 build 63 and any newer one*

### Steps/models to reproduce

- take a paper server 1.20.1 with higher build then build 62
- get the latest version of dynmap Dynmap_3.6-beta2
- you need to have a registered dynmap weblogin accout on your dynmap website
- try to login... you will see it fails

### Plugin and Datapack List

its just the dynmap plugin

### Paper version

1.20.1 Build 63 and newer

### Other
We tested this with latest build of paper 1.20.1 today and the issue is still there.
As solution to be able to use the dynmap weblog aggin is to downgraded to paper 1.20.1 build 62

**Comment by MiniDigger** [comment]
please talk to the dynmap people
if something in paper broke, they can more easily identify that and report back to us, but with the provided info we cant do anything.
build #62 was a significant upstream update, so its definetly not impossible that something got broken.

**Comment by JurgenKuyper** [comment]
Hi, dynmap support here, during login we check for banned players, and currently we get the following stacktrace;

```java

java.lang.ClassCastException: class com.destroystokyo.paper.profile.CraftPlayerProfile cannot be cast to class org.bukkit.craftbukkit.v1_20_R1.profile.CraftPlayerProfile (com.destroystokyo.paper.profile.CraftPlayerProfile and org.bukkit.craftbukkit.v1_20_R1.profile.CraftPlayerProfile are in unnamed module of loader java.net.URLClassLoader @446cdf90)
    at org.bukkit.craftbukkit.v1_20_R1.ban.CraftProfileBanList.isBanned(CraftProfileBanList.java:79) ~[paper-1.20.1.jar:git-Paper-64]
    at org.bukkit.craftbukkit.v1_20_R1.ban.CraftProfileBanList.isBanned(CraftProfileBanList.java:17) ~[paper-1.20.1.jar:git-Paper-64]
    at org.bukkit.craftbukkit.v1_20_R1.CraftOfflinePlayer.isBanned(CraftOfflinePlayer.java:111) ~[paper-1.20.1.jar:git-Paper-64]
    at org.dynmap.bukkit.DynmapPlugin$BukkitServer.isPlayerBanned(DynmapPlugin.java:321) ~[Dynmap-3.6-SNAPSHOT-spigot.jar:?]
    at org.dynmap.WebAuthManager.checkLogin(WebAuthManager.java:110) ~[Dynmap-3.6-SNAPSHOT-spigot.jar:?]
```
paper specific bug (doesn't happen with current Spigot)

That is what our dev found, hope it helps somewhat

**Comment by MiniDigger** [comment]
thanks, thats helpful, should be fixed by #9450 if I see this correctly 
so ill close this issue as a duplicate of #9455
